### PR TITLE
docs: update release notes

### PR DESCRIPTION
**Pull request by robert-zaremba**
## Description

Clarify Peggo instructions.

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -35,16 +34,24 @@ To run a node capable of supporting a liquidator, enable the query at compile ti
...
 See [Gravity Bridge](https://github.com/umee-network/Gravity-Bridge/blob/module/v1.5.3-umee-1/module/RELEASE_NOTES.md) Release Notes.
 
 ### Update notes
 
-Each validator must run:
+Each validator MUST:
 
-- Peggo (Gravity Bridge Orchestrator).
+- Peggo (Gravity Bridge Orchestrator) v1.0.x.
 - [Price Feeder](https://github.com/umee-network/umee/tree/main/price-feeder).
```
Comment (toteki) [inline RELEASE_NOTES.md:48]:
```suggestion
- Run Peggo (Gravity Bridge Orchestrator) v1.0.x.
- Run [Price Feeder](https://github.com/umee-network/umee/tree/main/price-feeder).
```

**Comment by RafilxTenfen** [comment]
We are using peggo [v0.4.1 ](https://github.com/umee-network/peggo/releases/tag/v0.4.1)
to release v1 I was hoping to get this in https://github.com/umee-network/peggo/pull/362

**Comment by robert-zaremba** [comment]
Will be merged just before the release.

**Review by toteki — APPROVED** [review]

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -35,16 +34,24 @@ To run a node capable of supporting a liquidator, enable the query at compile ti
...
-Each validator must run:
+Each validator MUST:
+
+- Run Peggo (Gravity Bridge Orchestrator) v1.0.x.
+- Run [Price Feeder](https://github.com/umee-network/umee/tree/main/price-feeder).
+- Update `config.toml` file by setting `mempool.version="v1"`. Ideally you should do it before the upgrade time, then at the upgrade switch binaries and start with the upgraded config:
 
-- Peggo (Gravity Bridge Orchestrator).
-- [Price Feeder](https://github.com/umee-network/umee/tree/main/price-feeder).
+  ```
```
Comment (toteki) [inline RELEASE_NOTES.md:51]:
```suggestion
  ```toml
```
lint

File: RELEASE_NOTES.md [hunk 3]
```diff
@@ -35,16 +34,24 @@ To run a node capable of supporting a liquidator, enable the query at compile ti
...
 
 See [Gravity Bridge](https://github.com/umee-network/Gravity-Bridge/blob/module/v1.5.3-umee-1/module/RELEASE_NOTES.md) Release Notes.
 
 ### Update notes
 
-Each validator must run:
+Each validator MUST:
+
+- Run Peggo (Gravity Bridge Orchestrator) v1.0.x.
+- Run [Price Feeder](https://github.com/umee-network/umee/tree/main/price-feeder).
```
Comment (toteki) [inline RELEASE_NOTES.md:48]:
```suggestion
- Run [Price Feeder](https://github.com/umee-network/umee/tree/main/price-feeder) v1.0.x
```

File: RELEASE_NOTES.md [hunk 4]
```diff
@@ -35,16 +34,24 @@ To run a node capable of supporting a liquidator, enable the query at compile ti
...
+Validators are expected to run Peggo and update the valiator set in Gravity smart contract.
 
 See [Gravity Bridge](https://github.com/umee-network/Gravity-Bridge/blob/module/v1.5.3-umee-1/module/RELEASE_NOTES.md) Release Notes.
 
 ### Update notes
 
-Each validator must run:
+Each validator MUST:
+
+- Run Peggo (Gravity Bridge Orchestrator) v1.0.x.
```
Comment (toteki) [inline RELEASE_NOTES.md:47]:
```suggestion
- Run Peggo (Gravity Bridge Orchestrator) v1.0.x
```

**Pull request merged**